docs/src/tutorial/next-steps.rst

```diff
@@ -0,0 +1,73 @@
+==========
+Next steps
+==========
+
+Congratulations! You have completed all of the tutorials for the ncov workflow. Read on for some next steps.
+
+.. contents:: Table of Contents
+   :local:
+
+.. _create-analysis-directory:
+
+Create your own analysis directory
+==================================
+
+On a web browser:
+
+1. `Sign up for a GitHub account <https://github.com/signup>`__ if you do not already have one.
+2. Create a repository from the ``ncov-tutorial`` template repository:
+
+   1. Go to https://github.com/nextstrain/ncov-tutorial.
+   2. Click **Use this template**.
+   3. Give your repository a name. We recommend ``my-ncov-analyses`` and will use that name in the following steps.
+   4. Click **Create repository from template**.
+
+In a command prompt:
+
+1. Go to the ``ncov/`` directory.
+2. Clone your new repository, replacing ``<username>`` with your own username:
+
+   .. code:: text
+
+      git clone https://github.com/<username>/my-ncov-analyses
+
+3. Read the next section to learn how to modify ``genomic-surveillance.yaml``.
+
+Modify the genomic surveillance workflow configuration
+======================================================
+
+Instead of an Idaho-focused workflow config, you can provide your own data for the ``custom_data`` input. Follow the same steps in the tutorial for GISAID download but select your own set of sequences and rename your ``metadata.tsv`` and ``sequences.fasta`` files accordingly.
+
+   .. note::
+
+      Workflow run time increases with the number of sequences, and the GISAID web interface has a maximum of 5,000 sequences per download.
+
+Then, use the following steps to customize names, titles, and context:
+
+1. Change the ``custom_data`` input filenames from ``idaho.metadata.tsv`` and ``idaho.sequences.fasta`` to your own.
+2. Change the regional input dataset from North America to an appropriate region for your custom focal data. :doc:`See the complete list of available URLs <../reference/remote_inputs>`.
+3. Rename the output dataset from ``idaho`` to your own. Note the name restrictions.
+4. Rename the subsampling scheme from ``idaho_scheme`` to your own. Note the name restrictions.
+5. Reword the output dataset title to your own.
+6. Rename the ``usa_context`` sample and update the ``query`` accordingly.
+
+.. warning::
+
+   File paths in the :term:`config files <config file>` must start with the :term:`analysis directory`. For example, in the tutorial:
+
+   .. code:: yaml
+
+      auspice_config: ncov-tutorial/auspice-config-custom-data.json
+
+   Now that you have created your own analysis directory, this must be modified, e.g.
+
+   .. code:: yaml
+
+      auspice_config: my-ncov-analyses/auspice-config-custom-data.json
+
+Additional resources
+====================
+
+- Learn more about genomic epidemiology. (TODO: link to Alli's gen epi book or some other resources)
```

Comment:
@jameshadfield @huddlej Anyone have the link for this?

Reply:
Maybe this?
> Add links to [CZI GenEpi](https://czgenepi.org/resources) and [CDC video](https://www.cdc.gov/amd/training/covid-19-gen-epi-toolkit.html) tutorials to our SARS-CoV-2 docs

Reply:
Those links are a great start! I don't know if Alli's book is out yet, but I will ask her.

Reply:
This was shared at office hours! https://alliblk.github.io/genepi-book/intro.html

Reply:
Done.